chinemelu/hotel-locator
# Consume endpoint that has Images and Integrate Endpoint with template

#### What does this PR do?
- Integrate images in the info bubble 

#### Description of Task to be completed?
- integrate image in info-bubble
- use different icon for hotel with stock images
- consume endpoint that have possible stock images for hotels

#### How should this be manually tested?
- clone the project
- run `npm install` and `npm run serve` respectively
- got to `/` and type an address
- check the map results and look out for images. Not every endpoint has images.

#### Any background context you want to provide?
- The previous endpoint didn't have access to images. This current endpoint has access to field that may or may not have images. 

#### Screenshots (if appropriate)
<img width="608" alt="Screen Shot 2020-07-15 at 11 46 02 PM" src="https://user-images.githubusercontent.com/26977462/87607730-e933e180-c6f5-11ea-84ca-6222ea2d8a89.png">

#### Questions:
NA
